plugins/modules/github_gpg_key.py

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright 2024, Austin Lucas Lake <53884490+austinlucaslake@users.noreply.github.com>
+# Based on community.general.github_key module by Ansible Project
+
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+module: github_gpg_key
+short_description: Manage GitHub GPG keys
+description:
+  - Creates, removes, or list GitHub GPG keys.
```

Comment:
"List" should not be mentioned as an operation. Listing should be done with `_info` modules.